### ISSUE TITLE
Button's outline has insufficient contrast against its adjacent background in "Enable telemetry dialogue"

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#855]
**Describe the bug**
The button's outline has insufficient contrast against its adjacent color

The following AI-windows modes have this issue:
- High Contrast White mode

**Expected behavior**
The button's outline should have a contrast ratio of at least 3:1 against its adjacent color

**Actual behavior**
The non-text contrast ratio of the button's outline against its adjacent color falls short of the required 3:1

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![2 1](https://user-images.githubusercontent.com/22068434/88341901-4a117a00-ccf3-11ea-92b9-d2316e3ce257.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
I think this is a false alarm. The button color is #808080, and the background color is #FFFFFF, which produces a contrast ratio of 3.949. The border is 3 pixels wide, and only the center is #808080 due to antialiasing. That said, there's an unbroken border all around the button, so I think we're OK.

**Comment by iamrafan** [comment]
@DaveTryon Sure, I also validated it in Canary Release: v1.1.1380.01
